scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -4059,70 +4687,1514 @@ async def run_gated(ctx: LiveQaContext) -> ProbeResult:
+                "leaked_raw_user_ids": sorted(set(leaked_ids)),
+            },
+        )
+    result.details["leaked_raw_user_ids"] = []
+    # Channel-ID arm (companion to qa_10i): a digest that surfaces raw C…/D…/G…
+    # conversation ids instead of conversation names has the same raw-entity
+    # hygiene failure as raw user ids.
+    leaked_conversation_ids = _raw_slack_conversation_ids_in_text(reply_text)
+    if leaked_conversation_ids:
+        return _result(
+            case_name,
+            False,
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)